### ISSUE TITLE
Sharing a build and install

### ISSUE THREAD

**Issue by kdorff** [redhawk668/ZX81-Composite#1]
You can close this. I just wanted to share.

I ordered the PCB from your repo and installed it in my TS1000 (US version of ZX81).  See https://imgur.com/gallery/2F53GVl

My thoughts, but no need to change anything.
* The position of the input/output pins nearly conflicts with the RCA jack (in the only orientation I could install the PCB). But it did work. Yay!
* I had trouble soldering the PCB to the can - getting the can to take solder. I realized LATER I needed to sand the can. I just soldered a ground wire to the bottom and brought that up (and then later sanded and soldered it to the insider of the can). My 22AWG solid core wire was a very tight fit to bring a ground from the bottom :)
* The video looks great!

My only suggestion MIGHT be to add a pin for ground, in case you don't want to solder it at the bottom to the can. But I should have realized to sand the inside of the can and it would have probably soldered just fine. And I should have used a fatter tip, maybe.

**Comment by BrettHallen** [comment]
Yep, I came across the same issue with the RCA jack last night in my TS1000!  In the ZX81 the RCA jack is at the other end of the modulator can so I believe this is what it was designed for. 

For soldering the ground to the can you could also try using some flux paste - this helps the solder adhere to the can.

Unfortunately I need to troubleshoot mine a bit more - the modulator can is somehow ending up at around 4.5V rather than ground ... but it's good to hear some someone that it does actually work!

[2022-08-15 09 29 07](https://user-images.githubusercontent.com/24483449/184558871-c1fe4325-b6f6-4235-a365-d1ef73a8b0c0.jpg)

Brett.

**Comment by kdorff** [comment]
I'm betting you bridged the legs on one of the transistors. Thanks for the flux paste recommendation. I have it and need to use it more.

**Comment by BrettHallen** [comment]
Ah huh, thanks, will have a proper look ... the legs are quite close together, aren't they! ;)  Your suggestion about an additional ground pin is good - the original strip board design has this, and this is also how I wire up the simple emitter-follower buffer I've been using before trying this out.  Of course, space is already at a premium on this board. 
Brett.

**Comment by BrettHallen** [comment]
Still not working.  Made up a 2nd board - no go.  I've found that there's two similar designs - the main difference being transistor - BC337 in this one and 2N2222 in the other.

**Comment by kdorff** [comment]
I've built this circuit and used it in two TS1000's. One that had a back porch (I omitted the 555) and one didn't. It worked great for both. 

Check your components and solder joints is all I can think. I first ran it outside of the can, running a ground wire to a bottom pad for testing. Make sure to take the video signal from pin 16 directly or use UK2 next to the RF can (but I wouldn't use USA3).

If you have a microscope, use that to check the solder joints on the transistor. This was initially my problem.

There was, from the original RF circuitry, a coil (?) and a capacitor soldered to the RCA Jack. I removed those before doing any testing.

I found the schematic [in this pdf](https://k1.spdns.de/Vintage/Sinclair/80/Sinclair%20ZX81/Service%20and%20Assembly%20Manual/ZX81%20Service%20and%20Assembly%20Manual%20(scans).pdf) to be helpful.
